src/Playwright/Helpers/JsonExtensions.cs

```diff
@@ -97,5 +97,25 @@ internal static JsonSerializerOptions GetNewDefaultSerializerOptions()
                     new JsonStringEnumMemberConverter(JsonNamingPolicy.CamelCase),
                 },
             };
+
+#nullable enable
+        internal static string? ToOptionalString(this JsonElement? element, string name)
```

Comment:
`GetOptionalString` ?

Reply:
`To` is more consistent with the rest of the helpers, like `ToString`, `ToObject`, etc.